SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/EventSearchAssetVoltageDisturbances.tsx

```diff
@@ -25,53 +25,66 @@ import React from 'react';
-    const [tableRows, setTableRows] = React.useState<JSX.Element[]>([]);
-    const seBrowserService = new SEBrowserService();
+    const [data, setData] = React.useState<IDisturbanceData[]>([]);
 
     React.useEffect(() => {
-        if (props.eventID >= 0) createTableRows(props.eventID);
-        return () => { };
+        if (props.eventID >= 0) loadDisturbancesData(props.eventID);
     }, [props.eventID]);
 
-    const createTableRows = (eventID: number) => {
-        seBrowserService.getEventSearchAsssetVoltageDisturbancesData(eventID).done((data) => {
-            const rows = data.map((d, i) => {
-                const style = { backgroundColor: d.IsWorstDisturbance ? 'lightyellow' : 'transparent' };
-                return (
-                    <tr key={i} style={style}>
-                        <td>{d.EventType}</td>
-                        <td>{d.Phase}</td>
-                        <td>{(d.PerUnitMagnitude * 100).toFixed(1)}</td>
-                        <td>{(d.DurationSeconds * 1000).toFixed(2)}</td>
-                        <td>{moment(d.StartTime).format('HH:mm:ss.SSS')}</td>
-                    </tr>
-                );
-            });
-            setTableRows(rows);
+    const loadDisturbancesData = (eventID: number) => {
+        seBrowserService.getEventSearchAsssetVoltageDisturbancesData(eventID).done((response) => {
+            const parsedData = response.map((d) => ({
+                ...d,
+                PerUnitMagnitude: (d.PerUnitMagnitude * 100).toFixed(1),
+                DurationSeconds: (d.DurationSeconds * 1000).toFixed(2),
+                StartTime: moment(d.StartTime).format('HH:mm:ss.SSS'),
+            }));
+            setData(parsedData);
         });
     };
 
     return (
         <div className="card">
             <div className="card-header">Voltage Disturbance in Waveform:</div>
             <div className="card-body">
-                <table className="table">
-                    <thead>
-                        <tr>
-                            <th>Disturbance Type</th>
-                            <th>Phase</th>
-                            <th>Magnitude (%)</th>
-                            <th>Duration (ms)</th>
-                            <th>Start Time</th>
-                        </tr>
-                    </thead>
-                    <tbody>{tableRows}</tbody>
-                </table>
+                <Table
+                    cols={[
+                        { key: 'EventType', field: 'EventType', label: 'Disturbance Type' },
+                        { key: 'Phase', field: 'Phase', label: 'Phase' },
+                        { key: 'PerUnitMagnitude', field: 'PerUnitMagnitude', label: 'Magnitude (%)' },
+                        { key: 'DurationSeconds', field: 'DurationSeconds', label: 'Duration (ms)' },
+                        { key: 'StartTime', field: 'StartTime', label: 'Start Time' },
```

Comment:
To add the formatting you can use:
```suggestion
                        { key: 'StartTime', field: 'StartTime', label: 'Start Time', content:(r) => moment(r.StartTime).format('HH:mm:ss.SSS')},
```